samples/app-anonymous-users/nodejs/ClientApp/package.json

```diff
@@ -13,10 +13,11 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.3",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "^2.1.3",
```

Comment:
Why we have reverted to use older version?

Reply:
fixed